File: fixtures/conformance/php/src/Shared.php
<?php
// Shared helpers with deliberately unique names for caller/callee assertions.
namespace Conf;

/** Uniquely named callee: exactly one caller (Conf::dispatch_conformance_request). */
function shared_unique_target(): void {}

/** High-degree callee: called by every Conf::fan_in_caller_* function. */
function record_conformance_event(string $label): void {}

File: fixtures/conformance/rust/src/shared.rs
//! Shared helpers with deliberately unique names for caller/callee assertions.

/// Uniquely named callee: exactly one caller (`service::dispatch_conformance_request`).
pub fn shared_unique_target() {}

/// High-degree callee: called by every `fan_in::fan_in_caller_*` function.
pub fn record_conformance_event(label: &str) {
    let _ = label;
}
